JudoPayDotNet/Models/SaveEncryptedCardModel.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+namespace JudoPayDotNet.Models
+{
+    /// <summary>
+    /// Data to save a card using a OneUseToken
+    /// </summary>
+    [DataContract]
+    // ReSharper disable UnusedMember.Global
+    public class SaveEncryptedCardModel : SaveCardModel
```

Comment:
If we are extending the SaveCardModel, doesn't this mean SaveEncryptedCardModel will also include a card number, CV2 etc. Will this not be confusing?

Reply:
Do we not have some other model which just contains the common attributes we need to extend, e.g. JudoID, consumerRef, address

Reply:
Here I followed the same logic as what we already do for for CheckEncryptedCardModel and RegisterEncryptedCardModel. I don't really want to change that as it would be breaking changes for anyone using the SDK.
Also, in GetSaveEncryptedCardModel() there is an example of making an instance of SaveEncryptedCardModel without providing card number or CV2 so as long as the documentation reflects that it should be okay.

Reply:
Ah OK. I checked out the history of those files, and they were both added by you, I would say their logic is wrong as well.
When we extend those models we get a card number in the token model, and this is clearly bad design as we don't want a card number in that model.
If it still works that's great and we can just ignore it, but design like this is why we have the receipt in apple pay model for example containing a token, and then we need to add documentation explaining why it's wrong, and our customers have to use confusing SDKs.